.envs/.local

```diff
@@ -0,0 +1,8 @@
+# DJANGO_READ_DOT_ENV_FILE=True
```

Comment:
Hadde det forresten vært mulig å løse dette på en måte som ikke involverer å måtte uncommente en linje i git-tracked fil? En smule slitsomt å passe på at man ikke committer den hele tiden hvis man skal ha inn CELERY_URL/AZURE_STORAGE_CONNECTION_STRING 😅

Reply:
Det med CELERY_URL er litt viktig faktisk, eller så må man få skrudd av Celery nå det ikke er en celery url så man unngår den evige forsøkingen på å koble til message broker:
![image](https://user-images.githubusercontent.com/31009729/117195090-26807180-ade5-11eb-8ce0-3098902e2808.png)

Reply:
Legger inn en default verdi for CELERY_URL 😉 Vil applikasjonen krasje en eller annen gang hvis vi ikke har AZURE_STORAGE_CONNECTION_STRING?

Reply:
Good. Den skal klare seg helt fint uten Azure_stringen frem til man prøver å laste opp en fil (naturligvis)